Mobile/iOS/Templates/Release_3.4.9/616-show_duplicated_trusted_federated.md

```diff
@@ -0,0 +1,23 @@
+###  Show duplicated users in trusted federated servers 
+
+#### Pr: https://github.com/owncloud/ios/pull/616 
+
+Devices:
+
+Server:
+
+Precondition: Set a server with federation app. Set a server as trusted to federated sharing.
+
+---
+
+ 
+| TestID | Test Case | Steps | Expected Result | Result | Related Comment |
+| :-----: | :------- | :---- | :-------------- | :----: | :-------------- |
+| 1 | Users in other server (with autoc.)| 1. Set in a remote trusted server two users with the same displayname<br> 2. Share a file/folder with that users| The autocompletion shows both distinguishing them (user@server)|||
+|**Regression**||||||
```

Comment:
I would add a test case with user with the character "@" in the username in local server, remote server, and remote trusted server